### ISSUE TITLE
Ability to use stack outputs / CloudFormation functions in environment?

### ISSUE THREAD

**Issue by wizardishungry** [svdgraaf/serverless-fargate-tasks#6]
I'm curious if removing this block would allow me to use stack outputs in my task defintions?
```js
 // any non-string value needs to be send through as json
```

**Comment by svdgraaf** [comment]
Hmmm, good catch. I think it would, you can use it for ref's then as well.
I'm not sure anymore why I added this specific if statement tbh.

I'll make this into a point release because it's potentially a breaking change.

**Comment by svdgraaf** [comment]
I just released a point release `0.4.0` with these changes.

**Comment by wizardishungry** [comment]
Are you sure you published it correctly –
```
$ curl 2> /dev/null https://registry.npmjs.org/serverless-fargate-tasks/-/serverless-fargate-tasks-0.4.0.tgz | gunzip - | grep -a JSON.stringify
            environment.push({"Name": key, "Value": JSON.stringify(value)})
```

**Comment by svdgraaf** [comment]
Interesting! Let me check :)

**Comment by svdgraaf** [comment]
Ok, I made a booboo. I just released 0.4.1 which should be okay now 😅 Sorry for that!

**Comment by wizardishungry** [comment]
No worries! 🥂
